components/support/rc_crypto/nss/src/util.rs

```diff
@@ -21,6 +22,24 @@ static NSS_INIT: Once = Once::new();
 #[cfg(feature = "keydb")]
 static NSS_PROFILE_PATH: OnceCell<String> = OnceCell::new();
 
+#[cfg(not(feature = "keydb"))]
+fn is_nss_initialized() -> bool {
+    NSS_INIT.is_completed()
+}
+
+#[cfg(feature = "keydb")]
+fn is_nss_initialized() -> bool {
+    NSS_INIT.is_completed() || NSS_PROFILE_PATH.get().is_some()
```

Comment:
this seems surprising to me - I guess I'd expect an `&&` here? Or possible an assertion if we expect both or neither to be true? Or even just a comment.